### PR TITLE
Feb 24 Refresh

### PR DESCRIPTION
**Pull request by dshanske**
Refreshes icons, adds code to generate a minimal CSS file that would be loaded in lieu of the full one, adds usernames for services Bridgy supports, fixes null bug.

### CODE REVIEW

**Comment by pfefferle** [comment]
There are some phpcs issues

**Comment by dshanske** [comment]
> There are some phpcs issues

Not exactly... there's an issue with duplicate keys in an autogenerated array I need to track down.

**Comment by dshanske** [comment]
Manually fixed for now. @pfefferle can you have a look

File: phpcs.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	<exclude-pattern>*/includes/class-connekt-plugin-installer.php</exclude-pattern>
 	<exclude-pattern>*/vendor/*</exclude-pattern>
 	<rule ref="PHPCompatibility"/>
-	<config name="testVersion" value="5.6-"/>
+	<config name="testVersion" value="7.2-"/>
```
Comment (pfefferle) [inline phpcs.xml:12]: WordPress requires 7.0 as min, shouldn't we use that?

**Review by pfefferle — APPROVED** [review]
LGTM

File: phpcs.xml (hunk 1 above)
Comment (dshanske) [reply phpcs.xml:12]: It is the debate of what version makes sense to support. They recommend 7.4, they set a minimum of 7.0. I sort of split the difference.

**Pull request merged**